js/src/dapps/registry/Names/actions.js

```diff
@@ -30,36 +31,23 @@ export const reserve = (name) => (dispatch, getState) => {
   const account = state.accounts.selected;
   const contract = state.contract;
   const fee = state.fee;
-
   if (!contract || !account) return;
+  name = name.toLowerCase();
```

Comment:
A line before the return break (which could do with some TLC), would be great here.